listenbrainz/webserver/static/js/profile.js

```diff
@@ -0,0 +1,687 @@
+'use strict';
```

Comment:
we have been using 2 space indentation in our JS and html code, should probably do the same here. (not a biggie, just mentioning)